css-positioning/absolute.css

```diff
@@ -0,0 +1,49 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+
+  position: absolute;
+  right: 30px;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+
+  position: absolute;
+  bottom: 5px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+
+  /* your code here */
+
+  position: absolute;
+  bottom: 0;
+  height: 300px;
+  width: 300px;
+}
```

Comment:
Instead of using `width` and `height` I want you to use position properties.

Also if you only set this to 300px, what happens if your image is larger? You have to change multiple rulesets to adjust for it.